environment/environment.go

```diff
@@ -398,6 +398,7 @@ func (m *Environment) Run() error {
 	}
 	if m.Cfg.JobImage != "" {
 		if err := m.Client.WaitForJob(m.Cfg.Namespace, "remote-test-runner", func(message string) {
+			fmt.Printf("Remote: %s\n", message)
```

Comment:
Where does this print get piped out to?

Reply:
Std out, an example of what the message look like when running a test with the remote runner:
`Remote: 6:01PM DBG Received block header Gas Price=10000000007 Hash=0x0982fda5d2cf50eda589f4836c1d62b1d3d10c6e125fe7ae2604dba6527889e2 NetworkName="Simulated Geth" Node=0 Number=508`
If I wrap this in a log.Info().Str(...).Msg(...) it creates string encoding problems since we end up with double encoding.